docs/next/modules/en/pages/troubleshooting/troubleshooting.adoc

```diff
@@ -0,0 +1,220 @@
+
+| capi-system
+| capi-controller-manager
+| Core Cluster API controller that reconciles main cluster resources
+
+| rke2-bootstrap-system
+| rke2-bootstrap-controller-manager
+| Manages the bootstrap configuration for RKE2 nodes
+
+| rke2-control-plane-system
+| rke2-control-plane-controller-manager
+| Handles the initialization and management of RKE2 control plane components
+
+| *-system
+| *-controller-manager
+| Handles the initialization and management of Infrastructure provider components
+
+| $-bootstrap-system
+| $-bootstrap-controller-manager
+| Manages the bootstrap configuration for custom providers
+
+| $-control-plane-system
+| $-control-plane-controller-manager
+| Handles the initialization and management of custom providers control plane components
+|===
+
+'*' - depending on the infrastructure provider in use
+'$' - depending on controlplane/bootstrap provider in use, e.g. kubeadm or rke2
+
+
+The best method to monitor logs from different pods is by using the following method.
+
+[source,bash]
+----
+kubectl krew install stern
+----
+
+Then you can tail logs from all pods in real time.
+
+[source,bash]
+----
+kubectl stern . -n rancher-turtles-system -n rke2-bootstrap-system -n rke2-control-plane-system -n capi-system
+----
+
+== How to manage Rancher Turtles and CAPI resources
+
+*List Rancher and Cluster API resources*
+
+Often users are asking how to deal with long list of different CRDs created by CAPI and Rancher, here is some idea how you can deal with it.
+
+. List all CAPI and Rancher related credentials
++
+[source,bash]
+----
+kubectl api-resources | grep 'cattle.io\|cluster.x-k8s.io'
+----
++
+. About resource you can get full explanation, what is it doing or what are configuration options
++
+[source,bash]
+----
+kubectl api-resources | grep clusterclass
+kubectl explain clusterclass.cluster.x-k8s.io
+----
++
+. Then you get a description about each field and configuration section 
++
+[source,bash]
+----
+kubectl explain clusterclass.spec
+kubectl explain clusterclass.spec.controlPlane
+kubectl explain clusterclasses.spec.controlPlane.machineInfrastructure
+----
+
+*List existing Rancher and Cluster API objects resources*
+
+The most common mistake is to use the `default` namespace in Kubernetes for your work. This will make your configuration really messy and difficult to manage and troubleshoot. It is highly recommended to use separate namespaces for deploying and creating CAPI downstream clusters.
+
+With separate namespaces, you can easily manage resources by following this method.
+
+. Install `get-all` and `lineage` plugins using `krew` for `kubectl`
++
+[source,bash]
+----
+kubectl krew install lineage
+kubectl krew install get-all
+----
++
+. Then list all existing resources in for example `capi-clusters` namespace where your downstream cluster configuration was deployed.
++
+[source,bash]
+----
+kubectl get-all -n capi-clusters
+----
```

Comment:
This is all better achieved by using `k9s`. It is also much better to share a `crust-gather` archive then use some third party plugins, as for the end user is better to work with a cluster as a whole. This includes `yaml` configurations, and resources outside of `-n capi-clusters`.